rocketpy/motors/Motor.py

```diff
@@ -276,10 +329,10 @@ def mass(self):
         Function
             Total propellant mass as a function of time.
         """
-        pass
+        return self.totalMassFlowRate.integralFunction() + self.propellantInitialMass
 
     @funcify_method("Time (s)", "mass dot (kg/s)", extrapolation="zero")
-    def massDot(self):
+    def totalMassFlowRate(self):
```

Comment:
This method evaluates what I like to call "theoretical" value of the `massFlowRate` based only on the thrust source, and is exactly the same as the old `massDot`. This is used by the `SolidMotor` class to compute the grains' `massFlowRate`. 

Things change regarding hybrid and liquid motors, because now there are two independent sources for the `massFlowRate`: the "theoretical" one that comes from the thrust curve, and the "empirical" one that comes from the Tanks' input data.

In a perfect world these values should be equivalent, however this is not the case for practical computations. Therefore each Motor subclass also has its own internal `massFlowRate` computation that works like this:

- `SolidMotor`: the `massFlowRate` is the `totalMassFlowRate`;
- `LiquidMotor`: the `massFlowRate` is the sum of the `Tank`'s `massFlowRate`. This value is independent of `totalMassFlowRate`;
- `HybridMotor`: the `massFlowRate` is the `totalMassFlowRate`, with the grain's `massFlowRate` being the difference between the `totalMassFlowRate` and the sum of the `Tank`'s `massFlowRate`.

Reply:
Great suggestion. I will improve the docstrings for the `massFlowRate` related methods and work on the variable exhaust velocity:

1. My first idea of implementing it would to be `override` the `Motor.exhaustVelocity` method in `LiquidMotor`. Any other suggestion?
2. Subtracting the pressure term might open its own can of worms: https://github.com/RocketPy-Team/RocketPy/pull/148 . There are many things to consider regarding who borrows data from whom. Furthermore the only way I can see of computing the nozzle exit pressure is from an user input, which is not bad, but is a calculation that either us or the user can make. Therefore I believe that this can be implemented without breaking changes and might be better off in the V1.1. Is this the pressure term you were talking about?

Reply:
I implemented the docstring changes and the variable exhaust velocity, as it can be seen on [73e584d](https://github.com/RocketPy-Team/RocketPy/pull/363/commits/73e584d863c15aa1bbb0d781b5ec0ea4fff1b5e3). Please let me know if it is working as expected and whether the docstring are clear enough.